src/Collector/RouterCollector.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Yiisoft\Yii\Debug\Collector;
+
+use Psr\Container\ContainerInterface;
+use Yiisoft\Router\RouteCollectorInterface;
+
+final class RouterCollector implements CollectorInterface
+{
+    use CollectorTrait;
+
+    private ContainerInterface $container;
+
+    public function __construct(ContainerInterface $container)
+    {
+        $this->container = $container;
+    }
+
+    public function getCollected(): array
+    {
+        return $this->container->get(RouteCollectorInterface::class)->getItems();
+    }
+
+    public function collect(...$payload): void
+    {
```

Comment:
We need a comment here explaining why we aren't collecting data in this method.

Reply:
> We need a comment here explaining why we aren't collecting data in this method.

Maybe we need to separate Collector onto Collector (that will use the method `collect()`) and "InfoProvider" (RouterInfoProvider, AppConfigInfoProvider, CachedSettingsInfoProvider(user's class)) that won't use the `collect()` method? What do you think?
in @yiiliveext PR the method `collect()` already dettached from `CollectorInterface`